HACKING.md

```diff
@@ -181,6 +186,8 @@ oc start-build --follow fedora-coreos-jenkins
 Once the Jenkins master image is built, Jenkins should start up (verify
 with `oc get pods`). Once the pod is marked READY, you should be able to
 login to the Jenkins UI at https://jenkins-fedora-coreos.$CLUSTER_URL/
+(an example URL looks like https://jenkins-fedora-coreos.127.0.0.1.nip.io/).
```

Comment:
Alternatively, you can mention `oc get route jenkins` which will print it out for you.